src/og/entity/GeoObjectHandler.js

```diff
@@ -370,6 +424,8 @@ class GeoObjectHandler {
             gl.bindBuffer(gl.ARRAY_BUFFER, this._vehicleVisibleBuffer[ti]);
             gl.vertexAttribPointer(a.aDispose, this._vehicleVisibleBuffer[ti].itemSize, gl.FLOAT, false, 0, 0);
 
+            gl.uniform1f(u.uUseTexture, this._textures[ti] ? 1 : 0);
```

Comment:
Remove uUseTexture

Reply:
this uniform used for mixing colors and textures